Source/mpqapi.cpp

```diff
@@ -545,47 +512,45 @@ BOOL mpqapi_open_archive(const char *pszArchive, BOOL hidden, int dwChar) // Ope
 // 65AB14: using guessed type char save_archive_open;
 // 679660: using guessed type char gbMaxPlayers;
 
-BOOLEAN mpqapi_parse_archive_header(_FILEHEADER *pHdr, int *pdwNextFileStart) // ParseMPQHeader
+BOOL mpqapi_parse_archive_header(_FILEHEADER *pHdr, int *pdwNextFileStart) // ParseMPQHeader

```

Comment:
Should this function be renamed to `ParseMPQHeader` based on the comment?

Reply:
FWIW, I added those comments based on matching functions from the [OpenD2](https://github.com/eezstreet/OpenD2/blob/master/Game/MPQ.cpp) project by @eezstreet